mmseg/models/segmentors/base.py

```diff
@@ -245,8 +245,12 @@ def show_result(self,
         seg = result[0]
         if palette is None:
             if self.PALETTE is None:
+                # Fix colors by setting random seed.
+                state = np.random.get_state()
```

Comment:
Can add more comments, refer to https://github.com/open-mmlab/mmdetection/blob/master/mmdet/core/visualization/image.py#L94-L96.

Reply:
Also, add this line in comments “The palette may be different in each iteration if not specified.”

Reply:
> Also, add this line in comments “The palette may be different in each iteration if not specified.”

But if set np.random.seed(42), the random palette will be fixed in each iteration.

Reply:
Of course. My point is an additional straightforward explanation in comments for this modification.